marshmallow_jsonschema/base.py

```diff
@@ -210,17 +213,21 @@ def _from_python_type(self, obj, field, pytype) -> typing.Dict[str, typing.Any]:
         metadata.update(field.metadata)
 
         for md_key, md_val in metadata.items():
-            if md_key in ("metadata", "name"):
+            if md_key in ("metadata", "name", PYTYPE_KEY):
                 continue
             json_schema[md_key] = md_val
 
-        if isinstance(field, fields.List):
-            json_schema["items"] = self._get_schema_for_field(obj, field.inner)
+        if pytype in (list, set, tuple):
+            json_schema["items"] = (
+                self._get_schema_for_field(obj, field.inner)
+                if isinstance(field, fields.List)
+                else {}
```

Comment:
This seems odd, wouldn't we still need to populate items for sets and tuples? Or a field that is just pretending to be a list but isn't actually inheriting from `fields.List`?